eqcorrscan/core/lag_calc.py

```diff
@@ -292,8 +297,18 @@ def xcorr_pick_family(family, stream, shift_len=0.2, min_cc=0.4,
                 if (tr.stats.station, tr.stats.channel) \
                         not in pick_stachans:
                     template_plot.remove(tr)
+            if plotdir is not None:
+                plotdir = "{plotdir}/{template_origin}/".format(plotdir=plotdir
+                , template_origin=event.resource_id.id[0:19])
+                if not os.path.isdir(plotdir):
+                    os.makedirs(plotdir)
+                savefile = "{plotdir}/{rid}.png".format(plotdir=plotdir, 
```

Comment:
You need to get rid of the space at the end of this line, and align the indents - try using a pep8 or flake8 plugin for your editor. A suggestion of what this should look like:
```python
savefile = "{plotdir}/{rid}.png".format(
    plotdir=plotdir, rid=event.resource_id.id)
```